extension/src/help/openWebHelp.ts

```diff
@@ -148,7 +148,7 @@ class WebHelpEntity {
 
 	static getDefaultHelpLink(): string {
 		let lang: string = WebHelpEntity.getLanguageUrlDomain();
-		let year: number = new Date().getFullYear() + 1;
+		let year: number = new Date().getFullYear();
```

Comment:
Can we have a fallback logic here? if 2022 doesn't exist, then we visit 2021?

Reply:
Its currently a blind open of a URL that is expected to exist. If we want to test it, then we'll have to do all kinds of HTTP GET work. Where does that end? If Current Year & CY + 1 both fail do we start trying other languages too?

In an email Hualin and I drew the conclusion it made sense to just target the current year to forever avoid this problem. It's not like the help really changes much from year to year and our JSON won't have the GUID's for the next releases new functions without doing the HelpAbstractionGenerator.exe process. If it next release doesn't exist, then we couldn't perform that collection step anyway. 

With that said, we could probably embed the target year into the webHelpAbstraction.json, but that is my only reasonable way to solve this dilemma without doing work that won't offer real benefits. If you want to go down that road, merge this PR and I can work on the various things associated with the C# project in a different one.